packages/grid/_modules_/grid/models/api/gridRowApi.ts

```diff
@@ -26,7 +26,7 @@ export interface GridRowApi {
    * Update any properties of the current set of GridRowData[].
    * @param updates
    */
-  updateRows: (updates: RowModelUpdate[]) => void;
+  updateRows: (updates: GridRowModelUpdate[]) => void;
```

Comment:
Is this a breaking change we should document?

Reply:
no it's not, it's just the grid prefix